TinyMCEPlugin.inc.php

```diff
@@ -21,7 +21,7 @@ class TinyMCEPlugin extends GenericPlugin {
 	 */
 	function register($category, $path, $mainContextId = null) {
 		if (parent::register($category, $path, $mainContextId)) {
-			if ($this->getEnabled($mainContextId)) {
+			if ($this->getEnabled()) {
```

Comment:
Any reason we're no longer providing `$mainContextId` upstream?

Reply:
Ah this looks like my mistake. `Plugin::getEnabled()` takes no arguments, but `LazyLoadPlugin::getEnabled()` does.

I've restored the argument there, but should these function signatures be synced?

Reply:
No, I think the discrepancy makes sense as it is, at least arguably. The `Plugin` implementation is just a trivial stub; it wouldn't use the parameter at all.